carmin-server/server/test/resources/test_executions.py

```diff
@@ -106,3 +106,27 @@ def test_post_invalid_model(self, test_config):
             data=POST_INVALID_MODEL)
         error = error_from_response(response)
         assert error == INVALID_MODEL_PROVIDED
+
+    def test_get_without_executions(self, test_config):
+        response = test_config.test_client.get(
+            '/executions', headers={
+                "apiKey": standard_user().api_key
+            })
+        assert json.loads(response.data) == []
+
+    def test_get_with_executions(self, test_config):
+        number_of_executions = 10
+        for _ in range(number_of_executions):
+            test_config.test_client.post(
+                '/executions',
+                headers={"apiKey": standard_user().api_key},
+                data=json.dumps(
+                    ExecutionSchema().dump(POST_VALID_EXECUTION).data))
```

Comment:
In case this is reused, could be moved to a fixture. I am also thinking about a few cases where I created an object using curly braces and put the model properties values instead of doing this. Could be an improvement for authenticate and register tests.

Reply:
Side comment about the fake execution data: For the INVALID_POST_EXECUTION, the custructor with None params should be used instead.

Reply:
I tried making it as a fixture, but wasn't exactly sure how to do it. The fixture would need access to `test_config`, and would technically need to make the `number_of_executions` variable visible to the test using the fixture. It could return it, but I thought this was somewhat hacky. Maybe you have a better idea on how to do this?

Reply:
In regards to the `POST_INVALID_MODEL` comment -- I tried that but it does not work: It forces me to marshal the `Execution` object before sending it out with the request, and marshmallow does not allow it since the model is not valid (missing required parameters).